### DISCUSSION TITLE
[Solved] Some titles not recognized

### DISCUSSION THREAD

**Discussion by swoop124** [sct/overseerr#2144] [General]
I have yesterday installed overseerr in a docker container. And i have 6 pretty big libraries in plex. Some titles are not recognized in overseerr. But they are all recognized correctly in Plex.

In the Log i found some Errormessages "Unable to find TMDb ID" but how is this possible when it is recognized in Plex?
Where can i add this or how can i assossiate a specific title?

thanks for any help

**Comment by swoop124** [comment]
I also just found out: 
I have for example one title in two seperate libraries but only the last one scanned is recognized in overseerr.
The other one is ignored.

**Comment by TheCatLady** [comment]
https://docs.overseerr.dev/support/faq#some-media-is-missing-from-overseerr-that-i-know-is-in-plex

**Comment by swoop124** [comment]
good morning,

thanks for the quick help, it helped.
I had to reload Metadata of the particular titles in Plex, than rescan in Overseerr and that's it.

its working great now.
